torchtext/vocab.py

```diff
@@ -68,21 +88,23 @@ def __init__(self, counter, max_size=None, min_freq=1, wv_path=None,
                 maximum. Default: None.
             min_freq: The minimum frequency needed to include a token in the
                 vocabulary. Default: 1.
-            wv_path: The path to the word vector file that will be loaded into
-                self.vectors, or None for no word vector file.
+            vectors: tuple or list specifying (directory, type, dimension)
+                of word vectors; None to go without word vectors
             lower: Whether to build a case-insensitive vocabulary.
             specials: The list of special tokens (e.g., padding or eos) that
                 will be prepended to the vocabulary in addition to an <unk>
                 token.
             fill_from_vectors: Whether to add to the vocabulary every token
-                for which a word vector is present in wv_path, even if the
-                token doesn't appear in the provided data.
+                for which a word vector specified by vectors is present
+                even if the token does not appear in the provided data.
+            unk_init: default to random initialization for unknown word vectors;
```

Comment:
It isn't quite "initialization for unknown word vectors" so much as "how to initialize the word vectors for words not in the pretrained word vector file"